cms/envs/bok_choy_static.py

```diff
@@ -0,0 +1,40 @@
+"""
+Settings for building static assets for Bok Choy tests.
```

Comment:
There should be more info in the docstring to explain to other developers why we are doing this, and also to remind our future selves (who will inevitably have forgotten the details.)